server/enterprise/metrics/metrics.go

```diff
@@ -1122,6 +1137,149 @@ func New(ps *platform.PlatformService, driver, dataSource string) *MetricsInterf
 	)
 	m.Registry.MustRegister(m.NotificationUnsupportedCounters)
 
+	m.ClientTimeToFirstByte = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: MetricsNamespace,
+			Subsystem: MetricsSubsystemClients,
+			Name:      "time_to_first_byte",
+			Help:      "Duration of when a browser starts to request a page from a server to when the server first starts to return data (milliseconds)",
```

Comment:
```suggestion
			Help:      "Duration from when a browser starts to request a page from a server to when the server first starts to return data (milliseconds)",
```

Reply:
Good point. I'll update the description slightly since it's when it starts to receive data